docs/pages/reference/operator-resources/resources.teleport.dev_accesslists.mdx

```diff
@@ -29,8 +29,10 @@ resource, which you can apply after installing the Teleport Kubernetes operator.
 |audit|[object](#specaudit)|audit describes the frequency that this Access List must be audited.|
 |description|string|description is an optional plaintext description of the Access List.|
 |grants|[object](#specgrants)|grants describes the access granted by membership to this Access List.|
+|member_of|[]string|member_of describes Access Lists that this Access List is an explicit member of.|
 |membership_requires|[object](#specmembership_requires)|membership_requires describes the requirements for a user to be a member of the Access List. For a membership to an Access List to be effective, the user must meet the requirements of Membership_requires and must be in the members list.|
 |owner_grants|[object](#specowner_grants)|owner_grants describes the access granted by owners to this Access List.|
+|owner_of|[]string|owner_of describes Access Lists that this Access List is an explicit owner of.|
```

Comment:
nit: `owner_of`/`member_of` are automatically managed backreferences, and cannot be set by the user, right?  If so, IMO they should either not be documented, or the docs should specify that they aren't user settable.

Reply:
true, would be good to append that to the desc here.